src/tracer/TraceExecutor.cc

```diff
@@ -63,23 +63,38 @@ executor::ExecuteAction TraceExecutor::onExecuteEvent(
     TraceEvent event{executeEvent};
     auto traceeInfo = rootTraceeInfo_->getProcess(event.executeEvent.pid);
     if (traceeInfo == nullptr) {
-        logger::debug(
-                "Got event for process ",
-                VAR(executeEvent.pid),
-                " which is not yet present in info tree, delaying");
+        auto delayedAction =
+                delayedPostCloneChildrenAction_.find(event.executeEvent.pid);
+        if (delayedAction != delayedPostCloneChildrenAction_.end()) {
+            executor::ExecuteAction action = executor::ExecuteAction::CONTINUE;
+            if (!executeEvent.exited && !executeEvent.killed) {
+                action = continueChildAfterClone(
+                        event.executeEvent.pid, delayedAction->second);
+            }
+            delayedPostCloneChildrenAction_.erase(delayedAction);
+            return action;
+        }
```

Comment:
Dlaczego w tej gałęzi ifa nie ma waitid?

Reply:
Bo tutaj od razu wykonuję `continueChildAfterClone` (w którym pójdzie `PTRACE_CONT`), albo proces ginie (i Executor sam odczyta jego status).

Reply:
Ok, ale nawet jak go wznowimy, to nie będzie i tak zakolejkowany event że on się zatrzymał zanim go wznowiliśmy, na który executor się natknie?

Reply:
Teraz to powinno być lepiej widać